disposable_email_blacklist.conf

```diff
@@ -45,17 +46,20 @@
 1st-forms.com
 1to1mail.org
 1zhuan.com
+2.emailfake.ml
```

Comment:
Not needed anymore as `emailfake.ml` will replace the need to declare `2.emailfake.ml`